spec/features/idv/in_person_spec.rb

```diff
@@ -222,6 +225,99 @@
     end
   end
 
+  context 'the user fails remote docauth, starts IPP, then navigates to how to verify and
+   resumes remote with successful images',
+          allow_browser_log: true do
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled).and_return(true)
+
+      visit_idp_from_sp_with_ial2(:oidc, **{ client_id: ipp_service_provider.issuer })
+      sign_in_via_branded_page(user)
+      complete_doc_auth_steps_before_document_capture_step(expect_accessible: true)
+      # Fail docauth
+      complete_document_capture_step_with_yml(
+        'spec/fixtures/ial2_test_credential_multiple_doc_auth_failures_both_sides.yml',
+        expected_path: idv_document_capture_url,
+      )
+      # begin in-person proofing
+      find(:button, t('in_person_proofing.body.cta.button'), wait: 10).click
+
+      complete_prepare_step
+      complete_location_step
+      complete_state_id_controller(user)
+
+      # Change mind and resume remote identity verification
+      visit idv_how_to_verify_url
+    end
+
+    it 'allows the user to successfully complete remote identity verification' do
+      # choose remote
+      click_on t('forms.buttons.continue_remote')
+      complete_hybrid_handoff_step
+      complete_document_capture_step(with_selfie: false)
+
+      complete_remote_idv_from_ssn(user)
+    end
+  end
+
+  context 'the user fails remote docauth, starts IPP, then navigates back to the submit images page
+  and resumes remote with successful images', allow_browser_log: true do
```

Comment:
Like my previous comment, what you have here is great code and this is a purely-optional stylistic difference suggestion.

These context names are really long. ❤️ that they're so expressive, but do you think it would read better if we could nest them, something like:

```ruby
context 'the user fails remote docauth and starts IPP', allow_browser_log: true do
  context 'then navigates to how to verify and resumes remote with successful images' do
    ...
  end

  context 'then navigates back to the submit images page and resumes remote with successful images' do
    ...
  end
end
```

If your thought isn't "Oh, I like that better!," please feel free to disregard this.

Reply:
See [a1ed99f](https://github.com/18F/identity-idp/pull/11948/commits/a1ed99fc573e6034c7762b018deda884e5d91921).